src/Pages/Articles.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState, useEffect } from "react";
+import { getAllArticles, getSingleArticle} from "../Api";
+import Card from "../Components/Card";
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState("")
+
+  useEffect(() => {
+    getAllArticles().then((articlesFromApi) => {
+      setArticles(articlesFromApi);
+      setIsLoading(false);
+    })
+    .catch(() => {
+      setIsError(true);
+      setIsLoading(false);
+    })
+  }, []);
+
+  const [searchArticle, setSearchArticle] = useState("");
+  useEffect(() => {
+    getSingleArticle(searchArticle).then((articleFromApi) => {
+    setSearchArticle(articleFromApi);
+    });
+  }, [searchArticle]);
```

Comment:
This is causing your error and it is not necessary to get a single article for this ticket, it's also done incorrectly.